edx_proctoring/utils.py

```diff
@@ -273,3 +269,18 @@ def verify_and_add_wait_deadline(context, exam, user_id):
         )
         return True
     return False
+
+
+def is_reattempting_exam(from_status, to_status):
+    """
+    Returns boolean if the user is trying to reattempt the exam.
+
+    given that user has started an exam, and he is kicked out due to low bandwidth or
+    closes the secure browser. Do not let the user restart the workflow from the start.
+    This is because the system only allow one attempt.
+    """
+    from_status_started = from_status == ProctoredExamStudentAttemptStatus.started
```

Comment:
check `ready_to_submit` state for it corresponds to an exam in progress.

Reply:
Sure.